### PR TITLE
Using requests instead of clickhouse-driver

### PR DESCRIPTION
**Pull request by StanislavZoldak**

### CODE REVIEW

File: src/__myproject__/_export/clickhouse_export.py [hunk 1]
```diff
@@ -37,20 +38,27 @@ def get_secrets(dbutils: DBUtils):
...
+    }
+
+# COMMAND ----------
+
+def execute_clickhouse_query(query: str):
+    response = requests.post(
+        f"http://{get_clickhouse_address.result['host']}:{get_clickhouse_address.result['port']}/",
+        params={"user": get_secrets.result["user"], "password": get_secrets.result["pass"], "default_format": "JSON"},
+        data=query)
+    return ast.literal_eval(response.content.decode("utf8") or "{}").get("data")
```
Comment (guderkar) [inline src/__myproject__/_export/clickhouse_export.py:55]: why ast necessary?

File: src/__myproject__/_export/clickhouse_export.py [hunk 2]
```diff
@@ -37,20 +38,27 @@ def get_secrets(dbutils: DBUtils):
...
 # COMMAND ----------
 
-@dp.notebook_function("%odapfeatures.clickhouse.host%", get_secrets)
-def clickhouse_connection(clickhouse_host: str, secrets: dict):
-    return ClickhouseClient(
-        clickhouse_host,
-        user=secrets["user"],
-        password=secrets["pass"]
-    )
+@dp.notebook_function("%odapfeatures.clickhouse.host%", "%odapfeatures.clickhouse.port%")
```
Comment (guderkar) [inline src/__myproject__/_export/clickhouse_export.py:41]: odapfeatures in template?

File: src/__myproject__/_export/clickhouse_export.py (hunk 1 above)
Comment (StanislavZoldak) [reply src/__myproject__/_export/clickhouse_export.py:55]: I'm parsing a bytes result and I found this to be a way to turn it into a dict. Is there a better way?

File: src/__myproject__/_export/clickhouse_export.py (hunk 2 above)
Comment (StanislavZoldak) [reply src/__myproject__/_export/clickhouse_export.py:41]: My bad, will fix.

**Review by guderkar — APPROVED** [review]

**Pull request merged**